alerts.js

```diff
@@ -0,0 +1,14 @@
+
+// ruleid:javascript-prompt
+var name = prompt('what is your name');
+// ruleid: javascript-alert
+alert('your name is ' + name);
```

Comment:
:exclamation:**Cycode**: SAST violation: 'artem-bitbucket-2'.
 
found alert() call; should this be in production code?
 
Severity: Medium
 <!-- be9d5f14-d91b-4ad8-8e43-661ecb125f0b -->